layouts/tech-partner-form/single.html

```diff
@@ -0,0 +1,177 @@
+    var formStart = document.getElementById("start-wrapper"); // initial page message
+    var startButton = document.getElementById("partner-form-start");
+    var firstStep = document.getElementById("partner-form-name");
+    var progressBarWrapper = document.getElementById("progress-bar-wrapper");
+    var stepButtons = document.getElementById("step-buttons");
+
+
+    var index = 0; // to be increased an decreased depending on button clicked
+    var progressBarValue = 8.333; // 100 divided by the 12 steps
+
+    var partner_form = {partner_form: true}
+
+    if(formStart && startButton && firstStep && progressBarWrapper && stepButtons) {
+      // remove beginning message and start process
+      startButton.onclick = function (event) {
+        formStart.setAttribute("style", "display:none;");
+        firstStep.setAttribute("style", "display:inherit;");
+        progressBarWrapper.setAttribute("style", "display:inherit;");
+        stepButtons.setAttribute("style", "display:inherit;");
+      }
+    }
+
+    nextButton.onclick = function (event) {
+      // parent[index] will always be current step
+      if(parent[index] && errElm && errElm[index]) {
+        // if required field is empty
+        if(parent[index].querySelector(".required") && parent[index].querySelector(".required").value == "") {
+          errElm[index].setAttribute("style", "display:block;");
+        } else {
+          var userEmailInput = parent[index].querySelector(".user-email");
+          // if email input does not include ampersand
+          if(userEmailInput && userEmailInput.value && !userEmailInput.value.includes("@")) {
+            errElm[index].setAttribute("style", "display:block;");
+          } else {
+            // next button successful click logic
+            parent[index].setAttribute("style", "display:none;");
+            if(backButton) {
+              backButton.setAttribute("style", "display:inline-block;");
+            }
+            if(progressBar) {
+              progressBar.innerHTML = index + 2
+              progressBarValue = progressBarValue + 8.333;
+              progressBar.setAttribute("style", `width: ${progressBarValue}%`);
+            }
+            index = index + 1
+            parent[index].setAttribute("style", "display:block;");
+            if(parent[index].classList.contains('partner-form-per-month')) {
+              nextButton.setAttribute("style", "display:none;");
+              submitButton.setAttribute("style", "display:inline-block;");
+            }
+          }
+        }
+      }
+    }
+
+    // toggle use case options
+    window.onmousedown = function (e) {
+      var el = e.target;
+      if (el.tagName.toLowerCase() == "option" &&
+          el.parentNode.hasAttribute("multiple") &&
+          el.className == "use-case") {
+
+        e.preventDefault();
+        // toggle selection
+        el.hasAttribute("selected") ? el.removeAttribute("selected") : el.setAttribute("selected", "")
```

Comment:
have you tried using el.toggleAttribute() here? Not sure it will work, but still